src/internet_identity/tests/integration/config.rs

```diff
@@ -21,7 +21,14 @@ fn should_retain_anchor_on_user_range_change() -> Result<(), CallError> {
             time_per_token_ns: 99,
             max_tokens: 874,
         }),
-        max_inflight_captchas: Some(456),
+        captcha_config: Some(CaptchaConfig {
+            max_unsolved_captchas: 788,
```

Comment:
Do the numbers mean anything? it was `456` before... I guess they don't, right?

Reply:
No, it's just different from the default, making sure that the values are not accidentally the same.